vizro-core/src/vizro/static/css/layout.css

```diff
@@ -161,8 +161,11 @@
   display: none;
 }
 
-#logo-dark,
-#logo-light {
+[data-bs-theme="dark"] #logo-dark {
+  display: inline;
+}
+
+[data-bs-theme="light"] #logo-light {
```

Comment:
not sure if this is relevant for this PR.

Can we refactor the styling for the #logo, display:inline can go to line 157.

Why do we need
```
#logo-dark,
#logo-light {
  display: none;
}
```

Reply:
I thought the same before but then remembered again why we have this here 😄 If you change it to above, then the hack of switching between a dark/light logo will not work anymore. The current code ensures that the light logo is only visible when the light theme is activated and the other way around. We don't want to always hide them.

Reply:
Will wait with merging until we've decided on the default